src/ce/real.hpp

```diff
@@ -0,0 +1 @@
+#include <ti/real>
```

Comment:
What is the purpose of this file?

Reply:
`<ti/real>` used to be `<real.hpp>`. Perhaps it should throw an error telling users to use the new name instead?

Reply:
It used to be **ti**real.hpp, didn't it?

Reply:
Yes, but I already fixed that.